src/test/java/ru/marina/tshop/users/UserControllerTest.java

```diff
@@ -0,0 +1,52 @@
+package ru.marina.tshop.users;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.mock.mockito.MockBean;
```

Comment:
Is Spring Boot's `org.springframework.boot.test.mock.mockito.MockBean` annotation actually necessary here?